src/python/doc/zbibliography.rst

```diff
@@ -0,0 +1,10 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+Bibliography
+------------
+
+.. bibliography:: ../../biblio/bibliography.bib
+    :style: unsrt
```

Comment:
Does `plain` work? Sorting the biblio looks a bit more natural.
`alpha` sure looks ugly...

Reply:
I used the `plain` but I did not really understand the difference with `unsrt`. Tell if it fits your will.

Reply:
It sorts the bibliography by alphabetical order of the first author, which is the usual way in this community.